linode/manager
# refactor: [M3-6229] - Remove /core/styles abstraction for tss-react codemod

## Description 📝
This is strictly a PR that updates imports.

We import styles from `src/components/core/styles`, but the codemod will not accurately be able to transform the file’s because it's expecting to find `@mui` as the path. We have to move away from this layer of abstraction and directly import them back into the files.

## Major Changes 🔄 
### Removed:
- `packages/manager/src/components/core/styles/index.ts`

### Changed:
- All JSS support has been moved to a legacy package `@mui/styles` with new package coming from `@mui/material/styles`.
- `useMediaQuery` now comes from `import useMediaQuery from '@mui/material/useMediaQuery';`
- `Theme`, `ThemeProvider`, and `useTheme` now come from `@mui/material/styles`

## How to test 🧪
- Run `yarn typecheck`
- Run locally, ensure pages still load and nothing looks obviously broken
- There are a few linting errors caused by previous work that we'll tackle in separate PR.
